readers/influxdb/messages_test.go

```diff
@@ -36,7 +36,7 @@ const (
 
 var (
 	v   float64 = 5
-	vs  string  = "a"
+	vs  string  = "value"
 	vb  bool    = true
```

Comment:
If `vd` is `dataValue`, then `vs` should be `stringValue`

Reply:
I've changed vd to "base64" and vs to "value", as it was originally

Reply:
I see below that `vd` is `dataValue`, so please use `stringValue` for `vs`

Reply:
There was a tiny bug in the code, which was causing the tests to fail when putting "stringValue", or making datavalue longer than stringvalue 😅. Silly mistake on my part; I've fixed it and the PR is ready now